docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -332,15 +332,39 @@ Default: `60`
 
 ##### --post_split_trigger_compaction_pool_max_threads
 
-The maximum number of threads allowed for post-split compactions (that is, compactions that remove irrelevant data from new tablets after splits).
+Deprecated. Use `full_compaction_pool_max_threads`.
+
+##### --post_split_trigger_compaction_pool_max_queue_size
+
+Deprecated. Use `full_compaction_pool_max_queue_size`.
+
+##### --full_compaction_pool_max_threads
+
+The maximum number of threads allowed for non-admin full compactions. This includes post-split compactions (compactions that remove irrelevant data from new tablets after splits) and scheduled full compactions.
 
 Default: `1`
 
-##### --post_split_trigger_compaction_pool_max_queue_size
+##### --full_compaction_pool_max_queue_size
+
+The maximum number of full compaction tasks that can be queued simultaneously. This includes post-split compactions (compactions that remove irrelevant data from new tablets after splits) and scheduled full compactions.
+
+Default: `200`
+
+##### --scheduled_full_compaction_frequency_hours
+
+The frequency with which full compactions should be scheduled on tablets. `0` indicates that the feature is disabled. Recommended value: `720` hours or greater (i.e. 30 days).
+
+Default: `0`
+
+##### --scheduled_full_compaction_jitter_factor_percentage
+
+Percentage of `scheduled_full_compaction_frequency_hours` to be used as jitter when determining full compaction schedule per tablet. Must be a value between `0` and `100`. Jitter is introduced to prevent many tablets from being scheduled for full compactions at the same time.
+
+Jitter will be deterministically computed when scheduling a compaction, between 0 and (frequency * jitter factor) hours. Once computed, the jitter will be subtracted from the intended compaction frequency to determined the tablet's next compaction time.
 
-The maximum number of post-split compaction tasks that can be queued simultaneously (compactions that remove irrelevant data from new tablets after splits).
+Example: If `scheduled_full_compaction_frequency_hours` is `720` hours (i.e. 30 days), and `scheduled_full_compaction_jitter_factor_percentage` is `33` percent, each tablet will be scheduled for compaction every `482` hours to `720` hours.
```

Comment:
I wonder if we can make it a bit clearer in the docs that the jitter factor implies a _range_, not an _absolute value_ to be applied to every tablet. I mean it seems obvious, but it also seems like an obvious support call we can avoid. ;-)

It would also be worth clarifying whether the jitter is calculated once per tablet and then remains consistent, or if each tablet's compaction interval will be randomly assigned every time.
```suggestion
For example, if `scheduled_full_compaction_frequency_hours` is 720 hours (30 days), and `scheduled_full_compaction_jitter_factor_percentage` is 33 percent, each tablet's compaction schedule can range from 482 to 720 hours.
```